Src/LinearSolvers/MLMG/AMReX_MLEBABecLap_3D_K.H

```diff
@@ -458,7 +461,7 @@ void mlebabeclap_adotx (Box const& box, Array4<Real> const& y,
             y(i,j,k,n) = alpha*a(i,j,k)*x(i,j,k,n) + (1.0/kappa) *
                 (dhx*(apxm*fxm - apxp*fxp) +
                  dhy*(apym*fym - apyp*fyp) +
-                 dhz*(apzm*fzm - apzp*fzp) - dhx*feb);
+                 dhz*(apzm*fzm - apzp*fzp) - dh*feb);
```

Comment:
It could be the case that the changes made here are equivalent to what we had previously when dx == dy == dz.  But it seems to me they are different because `dhx` and `dh` might different units in 3D.  So my suggestion is we revert all the changes in 3D.